templates/email/ChangePasswordEmail.ss

```diff
@@ -1,11 +1,11 @@
-<p><% _t('ChangePasswordEmail_ss.HELLO', 'Hi') %> $FirstName,</p>
+<p><%t ChangePasswordEmail_ss.HELLO 'Hi' %> $FirstName,</p>
 
 <p>
-	<% _t('ChangePasswordEmail_ss.CHANGEPASSWORDTEXT1', 'You changed your password for', 'for a url') %> $AbsoluteBaseURL.<br />
-	<% _t('ChangePasswordEmail_ss.CHANGEPASSWORDTEXT2', 'You can now use the following credentials to log in:') %>
+	<%t ChangePasswordEmail_ss.CHANGEPASSWORDTEXT1 'You changed your password for' is 'for a url' %> $AbsoluteBaseURL.<br />

```

Comment:
The intended use of the context info is to pull it into Transifex to help translators.